packages/integration-sdk-runtime/src/storage/FileSystemGraphObjectStore/indices.ts

```diff
@@ -75,13 +63,3 @@ export async function readGraphObjectFile<T>({
     });
   }
 }
-
-function isObjectFlushedEntityData(object: any): object is FlushedEntityData {
```

Comment:
`isObjectFlushedEntityData` and `isObjectFlushedRelationshipData` functions aren't needed. We fall back to an empty array when the property does not exist.